config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-kubelet-x-on-y.yaml

```diff
@@ -41,15 +41,15 @@ periodics:
           memory: "9000Mi"
           cpu: 2000m
 
-- name: ci-kubernetes-e2e-kubeadm-kinder-kubelet-1-20-on-1-21
+- name: ci-kubernetes-e2e-kubeadm-kinder-kubelet-1-20-on-latest
```

Comment:
I'm a little bit confuse by this test being changed by regeneration..
Also should this be 1.21-on-latest?

Reply:
Ohh might be this is actually a new test and the old one is now down in the file...

Reply:
Yup this is adding a new skew case.

Reply:
the diffs are noisy and difficult to review. thus the machinery to update them.

Reply:
the diffs depend on the order.

`ci-kubernetes-e2e-kubeadm-kinder-kubelet-1-20-on-1-21` below(or next) the old case
`ci-kubernetes-e2e-kubeadm-kinder-kubelet-1-20-on-latest` here is a new case

/lgtm
This Pr may need LGTM after rebase later.